.github/workflows/docker-buildx.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and publish docker image
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: ["Release"]
+    types:
+      - completed
+
+jobs:
```

Comment:
Should we remove `docker.yml` to merge this ?

Reply:
It can be kept a bit longer since they doesn't conflict in container name:
- docker.yml creates a container alephium/dev-alephium with  SNAPHOST versions
- this one creates alephium/alephium when a jar is released, i.e. not at every commits.